src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -1240,6 +1258,13 @@ private void AddFiles(Packaging.PackageBuilder builder, string itemType, string
                     continue;
                 }
 
+                if (IncludeSymbols &&
+                    SymbolPackageFormat == SymbolPackageFormat.Snupkg &&
+                    !string.Equals(Path.GetExtension(fullPath), ".pdb", StringComparison.OrdinalIgnoreCase))
```

Comment:
>StringComparison.OrdinalIgnoreCase [](start = 72, length = 34)

Should this be using `PathUtility.GetStringComparisonBasedOnOS`?

Reply:
this is an extension comparison, and extension in any casing would imply the same.